plugins/github-actions/README.md

```diff
@@ -92,3 +92,36 @@ integrations:
 - There is a limit of 100 apps for one OAuth client/token pair
 - The OAuth application must be at the GitHub organization level in order to display the workflows. If you do
   not see any workflows, confirm the OAuth application was created in the organization and not a specific user account.
+
+## Optional Workflow Runs Card View
+
+Github WorkflowRuns optional UI to show in Card view instead of table, with branch selection option
+
+![Card view example](./docs/card-view.png)
+
+1. Install the plugin dependency in your Backstage app package:
+
+```bash
+# From your Backstage root directory
+yarn add --cwd packages/app @backstage/plugin-github-actions
+```
+
+2. Add to the app `EntityPage` component:
+
+```tsx
+// In packages/app/src/components/catalog/EntityPage.tsx
+import {
+  EntityGithubActionsContent,
+  isGithubActionsAvailable,
+  ViewEnum
+} from '@backstage/plugin-github-actions';
```

Comment:
These instructions are already listed above, I'd say keep this section more concise by just highlighting the additional prop

Reply:
Updated and removed those instructions